README.md

```diff
@@ -57,9 +57,9 @@ Prerequisites
    $ sudo dnf install arm-none-eabi-newlib arm-none-eabi-gcc-cs
    ```
 
-2. Optional: libtock-c also includes support for building for ***RISC-V targets***.
-   These are not included by default since obtaining the toolchain can be
-   difficult (as of June 2020). You will need a `riscv64-unknown-elf` GCC
+2. Optional: libtock-c also includes support for building for ***RISC-V
+   targets***. These are not included by default since obtaining the toolchain
+   can be difficult (as of July 2022). You will need a `riscv64-unknown-elf` GCC
    toolchain that supports rv32 targets as well (i.e. is compiled with multilib
    support).
```

Comment:
I think that's not generally true. All RISC-V compilers are able to compile for `rv32i/ilp32` platforms, it's just the compiler intrinsics shipped as part of `libgcc` which can cause issues (which realistically I found to be an issue only for targets without hardware multiplication support, i.e. `rv32i`, see NixOS/nixpkgs#178140). Also we don't require a `riscv64-` toolchain, `riscv32-` and other tuples can work just fine.

So I agree with it being hard to acquire the correct toolchain, but perhaps we'd want to rephrase saying that we need a RISC-V toolchain able to target the respective chip's architecture?